python/cugraph/cugraph/tests/test_node2vec.py

```diff
@@ -172,14 +96,202 @@ def test_node2vec_invalid(
     for bad_depth in invalid_max_depths:
```

Comment:
Instead of having these `for loops` going over each parameter values,  why don't you just pass them to @pytest.mark.parametrize? isn't what it is meant for?

Reply:
Using @pytest.mark.parametrize would create tests for all combinations of invalid_max_depths, invalid_pqs, creating more tests than is necessary. This was a choice to reduce the number of tests without compromising coverage, since the pattern would be identical.

Reply:
Make sense